test/fw/ptl/utils/plugins/ptl_test_loader.py

```diff
@@ -208,7 +208,7 @@ def check_loadTestsFromNames(names, module=None):
                     tests_dir += [ptl_test_dir]
                 rv = old_loadTestsFromNames(tests_dir, module)
             else:
-                rv = old_loadTestsFromNames(names, module)
+                rv = old_loadTestsFromNames(tests_dir, module)
```

Comment:
you can remove else part and put old_loadTestsFromNames part outside of if condition